### PR TITLE
[1.x] Added fortify.domain and fortify.path to config stub

### PR DESCRIPTION
**Pull request by bilogic**
Added missing configuration options

### CODE REVIEW

**Comment by taylorotwell** [comment]
You can add them to your configuration if you want. If we add them here, the comment structure must be the same.

**Comment by bilogic** [comment]
Sorry what do you mean by comment structure must be the same?

**Comment by driesvints** [comment]
@bilogic your comments on the options don't follow the styling of the other existing ones.

**Comment by bilogic** [comment]
@driesvints ok let me try again, is there any reference for comments styling? Thank you.

**Comment by bilogic** [comment]
@driesvints would this meet the styling requirements? Thank you.
```PHP
    /*
     |--------------------------------------------------------------------------
     | Route domain
     |--------------------------------------------------------------------------
     |
     | By default, fortify routes are served on all domains.
     | To serve only a subdomain, specify a non-empty value.
     |
     */

    'domain' => '',

    /*
     |--------------------------------------------------------------------------
     | Route prefix
     |--------------------------------------------------------------------------
     |
     | By default, fortify routes do not have a prefix.
     | You are free to specify one here.
     |
     */

    'path' => '',
```

**Comment by driesvints** [comment]
Hi @bilogic. Our comment blocks are always three paragraphs long and the first line ends with three characters longer than the second line. The second line ends with two characters longer than the last one.

**Comment by bilogic** [comment]
@driesvints for a moment, I thought I was being trolled!
Pardon me, I don't mean to take up more of your time, but what is the rationale of that?

Meanwhile, I will try harder LOL.

**Comment by driesvints** [comment]
It's the code styling standard on all our config files of all our packages and the framework since more than 6 years already: https://github.com/laravel/laravel/blob/master/config/app.php

**Comment by bilogic** [comment]
Yes, I did check before replying earlier. 
But why? I'm honestly really curious about the thinking behind this requirement.
